crates/ruff_linter/src/rules/pyupgrade/rules/unnecessary_encode_utf8.rs

```diff
@@ -256,3 +263,78 @@ pub(crate) fn unnecessary_encode_utf8(checker: &Checker, call: &ast::ExprCall) {
         _ => {}
     }
 }
+
+fn string_contains_string_only_escapes(string: &StringLiteralValue, locator: &Locator) -> bool {
+    for fragment in string {
```

Comment:
Can we use the same terminology as in the ast: `literal` instead of inventing a new term.